backend/apps/slack/apps.py

```
@@ -24,13 +24,30 @@ class SlackConfig(AppConfig):
+        Args:
+            error (Exception): The exception raised.
+            body (dict): The payload of the Slack event that caused the error.
+            *args: Additional positional arguments.
+            **kwargs: Additional keyword arguments.
+        """
         logger.exception(error, extra={"body": body})
 
     @SlackConfig.app.use
-    def log_events(client, context, logger, payload, next):  # noqa: A002, ARG001
-        """Log Slack events."""
+    def log_events(client, context, logger, payload, next):
```

Ruff (0.8.2)

Unused function argument: `client`

(ARG001)